app/main/views/manage_users.py

```diff
@@ -61,6 +67,7 @@ def invite_user(service_id, user_id=None):
                 "views/user-already-team-member.html",
                 user_to_invite=user_to_invite,
             )
+        # TODO: Check for pending admin action
```

Comment:
Should we handle this TODO before merging?

Reply:
Good question.

If the incoming request is identical we could just silently succeed and not touch anything. But I guess we have two choices around how we should handle similar but not identical requests (e.g. inviting a user but with different permissions). We could:
- Automatically reject the existing pending request and create a new one
- Error out the creation of a new request and notify the user there's already something pending.

Personally I'm drawn to the former, as a non-admin (e.g. a service member with `manage_service` permission) still gets to action their intent.

Reply:
I've created a new status of 'invalidated' which all similar or identical pending requests will transition to before it creates a new one.